dnf/cli/cli.py

```diff
@@ -803,6 +803,11 @@ def configure(self, args, option_parser=None):
             e = '%s: %s' % (ucd(e.args[1]), repr(e.filename))
             logger.critical(_('Config error: %s'), e)
             sys.exit(1)
+        if opts.destdir is not None:
+            self.base.conf.destdir = opts.destdir
+            if not self.base.conf.downloadonly and opts.command != 'download':
```

Comment:
`if self.base.conf.downloadonly or opts.command == 'download'` and `if not self.base.conf.downloadonly and opts.command != 'download'` are logically equivalent, but the or variant is definitely more readable and easier to understand. Fixed.